README.md

```diff
@@ -1,112 +1,133 @@
 # codehub-readme-template
 
-# Summary of what a README.md is:
+# Summary of what a README file is:
 
-This is a template document meant to clearly and concisely convey what your application does and how it works. This is meant to advertise and encourage people to use your application.
+A README file contains important information about the associated source code, files, dependencies and etc., that make up an application. In order to leverage applications and their associated source code, members of the open source community rely on clear and concise READMEs.
 
-All of the following sections are recommended but order and naming of the sections is only encouraged. We only _require_ the Code.gov Registration Info section at the bottom of your README document.
+The following is a README template comprised of best practices for what information users expect to be included within a README.md file and how to organize for better user expereince. Pease feel free to copy and paste this template into your own repository's README space. 
 
-Note that ITS CodeHub has indicator badges that we place on the chiclet of your repository:
+# Organziational Outline:
+*Project Title
```

Comment:
Add a new line below `# Organizational Outline` and add a space between the asterisk and the items to fix the formatting